benchmarks/perception/a3_stereo_image_proc/description/inertial_macros.xacro

```diff
@@ -0,0 +1,40 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro" >
+
+    <!-- Specify some standard inertial calculations https://en.wikipedia.org/wiki/List_of_moments_of_inertia -->
+    <!-- These make use of xacro's mathematical functionality -->
+
+    <xacro:macro name="inertial_sphere" params="mass radius *origin">
```

Comment:
The code for the simulated robot and sensor models might be useful for other benchmarks in the future.  Could this code be moved to a separate package that can be imported by other benchmarks?

Reply:
Resolving this issue due to our discussion in the latest HAWG meeting